api/__tests__/routes/progress.test.ts

```diff
@@ -0,0 +1,1396 @@
+                            street: "Sint-Pietersnieuwstraat",
+                            zip_code: 9000,
+                        },
+                        deleted: false,
+                        description: "Description of building 2",
+                        id: 2,
+                        ivago_id: "ivago-2",
+                        name: "Building 2",
+                    },
+                    building_id: 2,
+                    deleted: false,
+                    departure: "2023-05-04T12:00:00.000Z",
+                    id: 2,
+                    images: [
+                        {
+                            deleted: false,
+                            description: "Description of progress image 2",
+                            id: 2,
+                            image: {
+                                id: 10,
+                                location: "IMGPROXY",
+                                path: "image.jpg",
+                                time: "1970-01-01T00:00:00.000Z",
+                                user_id: 1,
+                            },
+                            image_id: 10,
+                            progress_id: 2,
+                            type: "GARBAGE",
+                        },
+                    ],
+                    report: "Report 2",
+                    schedule: {
+                        day: "2023-05-04T12:00:00.000Z",
+                        deleted: false,
+                        end: "2023-05-04T12:20:00.000Z",
+                        id: 2,
+                        round: {
+                            description: "Description of round 2",
+                            id: 2,
+                            name: "Round 2",
+                        },
+                        round_id: 2,
+                        start: "2023-05-04T12:10:00.000Z",
+                        user: {
+                            address_id: 2,
+                            admin: false,
+                            date_added: "2023-05-04T12:00:00.000Z",
+                            deleted: false,
+                            email: "superstudent@trottoir.be",
+                            first_name: "Toon",
+                            id: 2,
+                            last_login: "2023-05-04T12:00:00.000Z",
+                            last_name: "De Superstudent",
+                            phone: "9876543210",
+                            student: false,
+                            super_student: true,
+                        },
+                        user_id: 2,
+                    },
+                    schedule_id: 2,
+                },
+            ];
+            await runner.get({
+                url: "/progress?deleted=true",
+                expectedData: expected,
+            });
+        });
+
+        test("SOFT DELETE /progress/:id (super-student)", async () => {
+            await runner.delete({ url: "/progress/1" });
```

Comment:
De testen worden sequentieel gerund. 

Deze test runt dus met hetzelfde auth niveau als de vorige test. Je kunt er niet van uitgaan dat de vorige test met SUPER_STUDENT niveau gerund is: in dit geval is dat zelfs ADMINISTRATOR. 

Twee mogelijke oplossingen: ofwel plaats een `beforeEach` hook die auth niveau op SUPER_STUDENT zet, ofwel zet de auth niveau binnen de test zelf.